### DISCUSSION TITLE
Reading several NFC chips

### DISCUSSION THREAD

**Discussion by vanitom** [capawesome-team/capacitor-plugins#174] [Q&A]
Hello,

I'm looking to use NFC to read several chips in a row.
Have you ever done this? 
How can it be implemented with promises?

Thanks in advance

**Comment by robingenz** [comment]
You just need to start a new scan session and listen to the [`addListener('nfcTagScanned', ...)`](https://capawesome.io/plugins/nfc/#addlistenernfctagscanned) listener. This listener is called every time a new nfc tag is scanned.
